tradewave/templates/tradewave/footer.html

```diff
@@ -22,7 +22,7 @@
               <div class="row">
                 <div class="6u 12u(mobile)">
                   <section>
-                   <h3 class="icon fa-envelope">
+                   <h3 class="icon fa-envelope-o" aria-hidden="true">
```

Comment:
@Urielar I tried this change locally and I did not see any difference. Does it change anything for you?